api/query/README.md

```diff
@@ -362,12 +362,13 @@ feature label, regardless of browsers.
 
     {"feature": [web-feature-name]}
 
-Where web-feature-name is a string, case-insensitive and matches the filename base
-for any of the .yml files in the
-[feature-group-definitions](https://github.com/web-platform-dx/web-features/tree/main/feature-group-definitions) directory.
```

Comment:
Thoughts on changing the old link to https://github.com/web-platform-dx/web-features/tree/main/features [1] ?

I wonder if we should point there instead because the WEB_FEATURES.yml files are scattered throughout the repository. Whereas [1] shows all the names in a list. 

But can be convinced to keep what you have.

Reply:
Yes, I think that is fine. I was operating under the assumption that it might be worthwhile to point users where this mapping is coming from, but it's easier to see what is searchable by looking at the features directory. 🙂